crates/router/src/cache.rs

```diff
@@ -63,6 +63,10 @@ impl Cache {
         let val = self.get(key)?;
         (*val).as_any().downcast_ref::<T>().cloned()
     }
+
+    pub async fn remove(&self, key: &str) {
```

Comment:
Hey @jeevaprakashdr can you also replace it `invalidate` with `remove` in the other places where `invalidate` is being used

Reply:
@dracarys18  ya can do. then in that case following will be the changes. is this okay?
*Url Endpoint will be renamed 
               **From**  http://localhost:8080/cache/invalidate/{key}
               **To** http://localhost:8080/cache/remove/{key}
*core/cache.rs method is renamed 
            **from** invalidate(key)
            **to** remove(key)

Reply:
It's fine to keep it as invalidate @jeevaprakashdr